the-collab-lab/tcl-19-smart-shopping-list
# Cu rb style additems component

## Description

This PR:
- adds styling to the AddItems component, using tailwind CSS framework according to the official figma design.
- adds styling to the Nav component using tailwind CSS framework according to the official figma design.

## Related Issue

closes #32 

## Acceptance Criteria

* Tailwind CSS is added as a dependency
* The Additems component style is implemented using the figma design and Tailwind CSS
* The Nav links Style is implemented using the figma design and Tailwind CSS

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### After

![Screenshot (114)](https://user-images.githubusercontent.com/57858821/108533357-ba827880-728d-11eb-9473-5557726d5226.png)
![Screenshot (115)](https://user-images.githubusercontent.com/57858821/108533364-bce4d280-728d-11eb-82bb-4ad4ad15d97d.png)
![Screenshot (116)](https://user-images.githubusercontent.com/57858821/108533402-c8d09480-728d-11eb-8928-26bd8fcb570f.png)



## Testing Steps / QA Criteria

* From your terminal, pull down this branch with "git pull origin cu-rb-style-additems-component" and check that branch out with "git checkout cu-rb-style-additems-component"
* start the server with "npm start"
* Navigate to the Additems Component to view the styles applied according this [design](https://www.figma.com/file/axGN4Wg2jZakmpoCn8S2zQ/Smart-shopping-list-Copy?node-id=160%3A74)
* Also observe the Nav styling for both Additems component and ItemsList component
